service/src/main/java/fish/focus/uvms/plugins/ais/service/AisService.java

```diff
@@ -124,6 +137,14 @@ private void connect() {
         connection.open(host, port, username, password);
     }
 
+    void fetchAssetList () {
+        SearchBranch searchBranch = new SearchBranch(true);
+        List<AssetDTO> assetDTOList =  assetClient.getAssetList(searchBranch);
+        for (AssetDTO assetDTO: assetDTOList) {
```

Comment:
We might want to empty out any entries in the set before adding the assets, i.e. `knownFishingVessels.clear();` since the freshly fetched vessels are what is now known to the system. Other than that, good job!